python/phylanx/ast/physl.py

```diff
@@ -1234,7 +1234,7 @@ def _Slice(self, node):
     def _Str(self, node):
         """class Str(s)"""
 
-        return '"' + node.s + '"'
+        return '"' + re.sub(r'(["|\\])',r'\\\1',node.s) + '"'
```

Comment:
Flake8 will fail as a couple of spaces are missing. In case no more changes are needed, here is the formatted version:
```
 return '"' + re.sub(r'(["|\\])', r'\\\1', node.s) + '"'
```